api/src/main/java/edu/cornell/mannlib/vitro/webapp/controller/freemarker/BaseSiteAdminController.java

```diff
@@ -199,12 +199,17 @@ protected ResponseValues processRequest(VitroRequest vreq) {
 
             String error = null;
             String explanation = null;
-            TBoxReasonerStatus status = ApplicationUtils.instance().getTBoxReasonerModule().getStatus();
-            if (!status.isConsistent()) {
-                error = "INCONSISTENT ONTOLOGY: reasoning halted.";
-                explanation = status.getExplanation();
-            } else if ( status.isInErrorState() ) {
-                error = "An error occurred during reasoning. Reasoning has been halted. See error log for details.";
+            try {
+                TBoxReasonerStatus status = ApplicationUtils.instance().getTBoxReasonerModule().getStatus();
+                if (!status.isConsistent()) {
+                    error = "INCONSISTENT ONTOLOGY: reasoning halted.";
+                    explanation = status.getExplanation();
+                } else if ( status.isInErrorState() ) {
+                    error = "An error occurred during reasoning. Reasoning has been halted. See error log for details.";
+                }
+            } catch (IllegalStateException e) {
+                error = "The inferencing engine is disabled.";
+                log.debug("Status of reasoner could not be determined.", e);
```

Comment:
It seems like the debug message would be something more like the `error`:
```
Status of reasoner could not be determined because the inferencing engine is disabled.
```